.travis.yml

```diff
@@ -72,7 +72,11 @@ script:
     fi
   - |
     if [[ $TRAVIS_EVENT_TYPE == "cron" ]]; then
-      bash scripts/travis-extract-l10n.sh
+      # Only run the extraction on "main" environment to avoid creating
+      # 8 pull requests for each tox environment.
+      if [[ $TOXENV == "main" ]]; then
```

Comment:
Is `main` the best job to link this to?  It runs at the end and is already the slowest (and prone to random failures we have to restart for). Would it run okay in the `codestyle` or `docs` jobs?

Reply:
I could even add an extra job for that… the thing is, nothing else *except* main is being executed because of the `if [[ $TRAVIS_EVENT_TYPE != "cron" ]]; then` in line 67 so it literally doesn't matter which env I choose as long as I chose something.